moveit_core/collision_detection/src/world.cpp

```diff
@@ -139,7 +139,8 @@ bool World::knowsTransform(const std::string& name) const
     for (const std::pair<const std::string, ObjectPtr>& object : objects_)
     {
       // if "object name/" matches start of object_id, we found the matching object
-      if (boost::starts_with(name, object.first) && name[object.first.length()] == '/')
+      // name.rfind is in service of removing the call to boost::starts_with and does the same thing
```

Comment:
Since you removed boost::starts_with, I don't think we need to mention this here. You could leave a comment on what rfind does, however.